lookup/src/pcs/kzg10.rs

```diff
@@ -0,0 +1,212 @@
+use ark_bls12_381::Fr;
+use ark_ff::PrimeField;
+use ark_serialize::CanonicalSerialize;
+use ark_std::test_rng;
+
+use kzg::commitment::KzgCommitment;
+use kzg::opening::KzgOpening as OtherKzgOpening;
+use kzg::scheme::KzgScheme;
+use kzg::srs::Srs;
+
+use crate::errors::Error;
+use crate::pcs::additive_homomorphic::AdditiveHomomorphicPCS;
+use crate::pcs::base_pcs::BasePCS;
+use crate::plookup::transcript_label::TranscriptLabel;
+use crate::poly::ExtraDensePoly;
+use crate::transcript::{ToBytes, TranscriptProtocol};
+use crate::types::{LookupProof, LookupProofTransferData, LookupVerifyTransferData, Poly};
+
+impl ToBytes for KzgCommitment {
+    fn to_bytes(&self) -> Vec<u8> {
+        let mut bytes = vec![];
+        self.inner().serialize_uncompressed(&mut bytes).expect("");
+        bytes
+    }
+}
+pub type KzgField = Fr;
+
+/// This struct represents KZG scheme using BLS12_381
+pub struct KZG12_381 {
+    scheme: KzgScheme,
+}
+
+/// This struct represents extra proof that used to verify in `Plookup` using KZG
+pub struct PlookupKZGProof<Commitment> {
+    /// Commitment of aggregated quotient polynomial that evaluated at `evaluation_challenge`
+    pub agg_quotient_witness: Commitment,
```

Comment:
The name `agg_quotient_witness` is misleading. You should change it.

```suggestion
agg_quotient_commitment
```